src/components/sidebar/sidebar.css

```diff
@@ -1,23 +1,37 @@
 .sidebar {
   padding: 1em;
-  overflow-x: hidden;
+  overflow: auto;
 
   font-size: 11px;
   background: var(--grey-10);
   border-top: 1px solid var(--grey-30);
+
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 2px 5px;
+  align-content: start; /* the grid isn't vertically stretched */
+
+  line-height: 1.5;
+}
+
+.sidebar-titlegroup, .sidebar-title2, .sidebar-title3 {
+  grid-column-start: span 2;
+  min-width: 0; /* With this, the cell's minimum size is 0 instead of min-content */
 }
 
 .sidebar-titlegroup {
   margin: 0 0 1em 0;
-  line-height: 1.2;
-  min-height: 8em; /* This allows for a more stable sidebar when navigating the call tree */
+  line-height: unset;
 }
 
 .sidebar-title, .sidebar-subtitle {
   margin: 0;
   padding: 0;
   font-weight: normal;
 
+  overflow: auto;
+  max-height: 3.7em; /* ~2.5 lines */
```

Comment:
I'm not very happy about the effect we get here; on Linux this is a bit ugly. Maybe it's better on Mac with your hover scrollbars though.
This was a request from @digitarald.

Reply:
This does not feel nice even with the hover scrollbars. My preference would be to not cut off the text, since this area can scroll.

Reply:
If we don't set a max height, then the sidebar's data will jump a lot when navigating. When trying it I found it distracting when looking for something special. But maybe navigating while looking at the sidebar isn't something so useful? What do you think @digitarald ?

Reply:
On the version currently in master, I've set a min-height on the header so that it doesn't jump while retaining the full text, but this leaves a disgraceful space when the header is small...

Reply:
Alternative idea: you could make it a single line, with `text-overflow: ellipsis`, and then put the full value in a tooltip and/or make it copyable by selection. You could even use a text input for this:
`<input type="text" value="Some very long value" style="border:0;background:none;text-overflow:ellipsis" readonly>`
Text inputs are scrollable horizontally in Firefox and don't show a scrollbar.

Reply:
Or the single line with ellipsis and tooltip, but contenteditable so that you can select and copy it too

Reply:
That'd work, too.